templates/galaxy/config/object_store_conf.xml.j2

```diff
@@ -1,32 +1,37 @@
 <?xml version="1.0"?>
 <object_store type="distributed" id="primary" order="0">
     <backends search_for_missing="false">
-        <backend id="files1" type="disk" weight="0" store_by="id">
+        <backend id="secondary" type="disk" weight="0" store_by="id">
+            <files_dir path="/data/0/galaxy_db/files"/>
+            <extra_dir type="temp" path="/data/2/galaxy_db/tmp"/>
+            <extra_dir type="job_work" path="/data/jwd/main"/>
+        </backend>
+        <backend id="files1" type="disk" weight="0" store_by="id" name="University of Freiburg storage (NetApp FabricPool, id=files1)">
             <files_dir path="/data/1/galaxy_db/files"/>
             <extra_dir type="temp" path="/data/2/galaxy_db/tmp"/>
             <extra_dir type="job_work" path="/data/jwd/main"/>
         </backend>
-        <backend id="files2" type="disk" weight="0" store_by="id">
+        <backend id="files2" type="disk" weight="0" store_by="id" name="University of Freiburg storage (NetApp FabricPool, id=files2)">
             <files_dir path="/data/2/galaxy_db/files"/>
             <extra_dir type="temp" path="/data/2/galaxy_db/tmp"/>
             <extra_dir type="job_work" path="/data/jwd/main"/>
         </backend>
-        <backend id="files3" type="disk" weight="0" store_by="id">
+        <backend id="files3" type="disk" weight="0" store_by="id" name="University of Freiburg storage (NetApp FabricPool, id=files3)">
             <files_dir path="/data/3/galaxy_db/files"/>
             <extra_dir type="temp" path="/data/2/galaxy_db/tmp"/>
             <extra_dir type="job_work" path="/data/jwd/main"/>
         </backend>
-        <backend id="files4" type="disk" weight="0" store_by="id">
+        <backend id="files4" type="disk" weight="0" store_by="id" name="University of Freiburg storage (NetApp FabricPool, id=files4)">
```

Comment:
Do we want to keep the `name=...` attribute consistent for all the backends? for some we are not including it in the below.

Reply:
For now yes, I have a cleanup PR that comes next and I will fix a few of those there and maybe adjust the badges.